### PR TITLE
[WIP] Dockerize the test environment

### PR DESCRIPTION
**Pull request by Fryguy**
@psav I'm just making a WIP PR so you can see how far we've gotten, so we can collaborate on getting this runnable by dev.

Includes #5965 

Some notes:
- I don't want to set up BZ creds, because it makes no sense to me to need creds for Bugzilla to run tests, so I've commented out the places where it complains.  Strangely, I don't have to comment out *everything* and I don't understand why.

Current status:
`9 failed, 230 passed, 5 skipped, 225 deselected, 253 warnings, 14 error in 592.18 seconds`

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 98d0c55f809658d93654102b145369789c980343:

`cfme/tests/ansible/test_embedded_ansible_actions.py`:
- :bangbang: Line 25:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 136:1: **E265** *block comment should start with '# '*

`cfme/tests/ansible/test_embedded_ansible_services.py`:
- :bangbang: Line 24:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/automate/test_buttons.py`:
- :bangbang: Line 137:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_class.py`:
- :bangbang: Line 97:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_vmware_methods.py`:
- :bangbang: Line 81:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_object_tag_visibility.py`:
- :bangbang: Line 93:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_timelines.py`:
- :bangbang: Line 131:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_events.py`:
- :bangbang: Line 37:1: **E265** *block comment should start with '# '*
- :bangbang: Line 38:1: **E265** *block comment should start with '# '*
- :bangbang: Line 41:1: **E265** *block comment should start with '# '*
- :bangbang: Line 42:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_rest_providers.py`:
- :bangbang: Line 208:1: **E265** *block comment should start with '# '*
- :bangbang: Line 230:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_retirement.py`:
- :bangbang: Line 194:2: **E999** *SyntaxError: invalid syntax*
- :bangbang: Line 194:3: **E227** *missing whitespace around bitwise or shift operator*
- :bangbang: Line 194:7: **E225** *missing whitespace around operator*
- :bangbang: Line 196:3: **E225** *missing whitespace around operator*
- :bangbang: Line 196:5: **E225** *missing whitespace around operator*
- :bangbang: Line 196:7: **E225** *missing whitespace around operator*
- :bangbang: Line 197:1: **E265** *block comment should start with '# '*
- :bangbang: Line 199:7: **E225** *missing whitespace around operator*
- :heavy_exclamation_mark: Line 199:21: **W604** *backticks are deprecated, use 'repr()'*
- :bangbang: Line 233:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_vm_ownership.py`:
- :bangbang: Line 19:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/configure/test_access_control.py`:
- :bangbang: Line 254:1: **E265** *block comment should start with '# '*
- :bangbang: Line 255:1: **E265** *block comment should start with '# '*
- :bangbang: Line 865:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_default_filters.py`:
- :bangbang: Line 60:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 187:1: **E265** *block comment should start with '# '*
- :bangbang: Line 215:1: **E265** *block comment should start with '# '*
- :bangbang: Line 235:1: **E265** *block comment should start with '# '*
- :bangbang: Line 237:1: **E265** *block comment should start with '# '*
- :bangbang: Line 289:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_register_appliance.py`:
- :bangbang: Line 116:1: **E265** *block comment should start with '# '*
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_tag.py`:
- :bangbang: Line 202:5: **E265** *block comment should start with '# '*

`cfme/tests/containers/test_containers_summary.py`:
- :bangbang: Line 26:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 44:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_alerts.py`:
- :bangbang: Line 35:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 36:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_compliance.py`:
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_rest_control.py`:
- :bangbang: Line 156:5: **E265** *block comment should start with '# '*
- :bangbang: Line 189:5: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_config_management.py`:
- :bangbang: Line 14:15: **E114** *indentation is not a multiple of four (comment)*
- :bangbang: Line 14:15: **E116** *unexpected indentation (comment)*
- :bangbang: Line 14:15: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_instance_analysis.py`:
- :bangbang: Line 610:1: **E265** *block comment should start with '# '*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by psav** [comment]
woohoo @Fryguy is here

**Comment by dajoRH** [comment]
Lint report for commit 473b411bafe09780296ffa832ae43f0a47b74f31:

`cfme/tests/ansible/test_embedded_ansible_actions.py`:
- :bangbang: Line 25:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 136:1: **E265** *block comment should start with '# '*

`cfme/tests/ansible/test_embedded_ansible_services.py`:
- :bangbang: Line 24:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/automate/test_buttons.py`:
- :bangbang: Line 137:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_class.py`:
- :bangbang: Line 97:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_vmware_methods.py`:
- :bangbang: Line 81:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_object_tag_visibility.py`:
- :bangbang: Line 93:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_timelines.py`:
- :bangbang: Line 131:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_events.py`:
- :bangbang: Line 37:1: **E265** *block comment should start with '# '*
- :bangbang: Line 38:1: **E265** *block comment should start with '# '*
- :bangbang: Line 41:1: **E265** *block comment should start with '# '*
- :bangbang: Line 42:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_rest_providers.py`:
- :bangbang: Line 208:1: **E265** *block comment should start with '# '*
- :bangbang: Line 230:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_retirement.py`:
- :bangbang: Line 194:1: **E265** *block comment should start with '# '*
- :bangbang: Line 228:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_vm_ownership.py`:
- :bangbang: Line 19:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/configure/test_access_control.py`:
- :bangbang: Line 254:1: **E265** *block comment should start with '# '*
- :bangbang: Line 255:1: **E265** *block comment should start with '# '*
- :bangbang: Line 865:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_default_filters.py`:
- :bangbang: Line 60:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 187:1: **E265** *block comment should start with '# '*
- :bangbang: Line 215:1: **E265** *block comment should start with '# '*
- :bangbang: Line 235:1: **E265** *block comment should start with '# '*
- :bangbang: Line 237:1: **E265** *block comment should start with '# '*
- :bangbang: Line 289:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_register_appliance.py`:
- :bangbang: Line 116:1: **E265** *block comment should start with '# '*
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_tag.py`:
- :bangbang: Line 202:5: **E265** *block comment should start with '# '*

`cfme/tests/containers/test_containers_summary.py`:
- :bangbang: Line 26:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 44:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_alerts.py`:
- :bangbang: Line 35:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 36:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_compliance.py`:
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_rest_control.py`:
- :bangbang: Line 156:5: **E265** *block comment should start with '# '*
- :bangbang: Line 189:5: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_config_management.py`:
- :bangbang: Line 14:15: **E114** *indentation is not a multiple of four (comment)*
- :bangbang: Line 14:15: **E116** *unexpected indentation (comment)*
- :bangbang: Line 14:15: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_instance_analysis.py`:
- :bangbang: Line 610:1: **E265** *block comment should start with '# '*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d51a49a0342a5f516025bc31ce18bdff9e7eb36c:

`cfme/tests/ansible/test_embedded_ansible_actions.py`:
- :bangbang: Line 25:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 136:1: **E265** *block comment should start with '# '*

`cfme/tests/ansible/test_embedded_ansible_services.py`:
- :bangbang: Line 24:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/automate/test_buttons.py`:
- :bangbang: Line 137:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_class.py`:
- :bangbang: Line 97:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_vmware_methods.py`:
- :bangbang: Line 81:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_object_tag_visibility.py`:
- :bangbang: Line 93:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_timelines.py`:
- :bangbang: Line 131:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_events.py`:
- :bangbang: Line 37:1: **E265** *block comment should start with '# '*
- :bangbang: Line 38:1: **E265** *block comment should start with '# '*
- :bangbang: Line 41:1: **E265** *block comment should start with '# '*
- :bangbang: Line 42:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_rest_providers.py`:
- :bangbang: Line 208:1: **E265** *block comment should start with '# '*
- :bangbang: Line 230:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_retirement.py`:
- :bangbang: Line 194:1: **E265** *block comment should start with '# '*
- :bangbang: Line 228:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_vm_ownership.py`:
- :bangbang: Line 19:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/configure/test_access_control.py`:
- :bangbang: Line 254:1: **E265** *block comment should start with '# '*
- :bangbang: Line 255:1: **E265** *block comment should start with '# '*
- :bangbang: Line 865:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_default_filters.py`:
- :bangbang: Line 60:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 187:1: **E265** *block comment should start with '# '*
- :bangbang: Line 215:1: **E265** *block comment should start with '# '*
- :bangbang: Line 235:1: **E265** *block comment should start with '# '*
- :bangbang: Line 237:1: **E265** *block comment should start with '# '*
- :bangbang: Line 289:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_register_appliance.py`:
- :bangbang: Line 116:1: **E265** *block comment should start with '# '*
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_tag.py`:
- :bangbang: Line 202:5: **E265** *block comment should start with '# '*

`cfme/tests/containers/test_containers_summary.py`:
- :bangbang: Line 26:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 44:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_alerts.py`:
- :bangbang: Line 35:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 36:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_compliance.py`:
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_rest_control.py`:
- :bangbang: Line 156:5: **E265** *block comment should start with '# '*
- :bangbang: Line 189:5: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_config_management.py`:
- :bangbang: Line 14:15: **E114** *indentation is not a multiple of four (comment)*
- :bangbang: Line 14:15: **E116** *unexpected indentation (comment)*
- :bangbang: Line 14:15: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_instance_analysis.py`:
- :bangbang: Line 610:1: **E265** *block comment should start with '# '*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cec6997ff5e209422be2e141913ff417bd563bb9:

`cfme/tests/ansible/test_embedded_ansible_actions.py`:
- :bangbang: Line 25:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 136:1: **E265** *block comment should start with '# '*

`cfme/tests/ansible/test_embedded_ansible_services.py`:
- :bangbang: Line 24:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/automate/test_buttons.py`:
- :bangbang: Line 137:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_class.py`:
- :bangbang: Line 97:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_vmware_methods.py`:
- :bangbang: Line 81:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_object_tag_visibility.py`:
- :bangbang: Line 93:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_timelines.py`:
- :bangbang: Line 131:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_events.py`:
- :bangbang: Line 37:1: **E265** *block comment should start with '# '*
- :bangbang: Line 38:1: **E265** *block comment should start with '# '*
- :bangbang: Line 41:1: **E265** *block comment should start with '# '*
- :bangbang: Line 42:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_rest_providers.py`:
- :bangbang: Line 208:1: **E265** *block comment should start with '# '*
- :bangbang: Line 230:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_retirement.py`:
- :bangbang: Line 194:1: **E265** *block comment should start with '# '*
- :bangbang: Line 228:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_vm_ownership.py`:
- :bangbang: Line 19:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/configure/test_access_control.py`:
- :bangbang: Line 254:1: **E265** *block comment should start with '# '*
- :bangbang: Line 255:1: **E265** *block comment should start with '# '*
- :bangbang: Line 865:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_default_filters.py`:
- :bangbang: Line 60:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 187:1: **E265** *block comment should start with '# '*
- :bangbang: Line 215:1: **E265** *block comment should start with '# '*
- :bangbang: Line 235:1: **E265** *block comment should start with '# '*
- :bangbang: Line 237:1: **E265** *block comment should start with '# '*
- :bangbang: Line 289:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_register_appliance.py`:
- :bangbang: Line 116:1: **E265** *block comment should start with '# '*
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_tag.py`:
- :bangbang: Line 202:5: **E265** *block comment should start with '# '*

`cfme/tests/containers/test_containers_summary.py`:
- :bangbang: Line 26:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 44:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_alerts.py`:
- :bangbang: Line 35:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 36:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_compliance.py`:
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_rest_control.py`:
- :bangbang: Line 156:5: **E265** *block comment should start with '# '*
- :bangbang: Line 189:5: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_config_management.py`:
- :bangbang: Line 14:15: **E114** *indentation is not a multiple of four (comment)*
- :bangbang: Line 14:15: **E116** *unexpected indentation (comment)*
- :bangbang: Line 14:15: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_instance_analysis.py`:
- :bangbang: Line 610:1: **E265** *block comment should start with '# '*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ADD docker-assets/entrypoint /entrypoint
+
+RUN touch .yaml_key
+RUN mkdir -p ../cfme-qe-yamls/complete/
+
+### DELETE ME ###
+# Temporary manual run of the quickstart, so these can build separate layers
+# in the image, and not require rebuilding one giant layer
+RUN ../cfme_venv/bin/pip install -U pip wheel setuptools_scm docutils
+RUN ../cfme_venv/bin/pip install -r requirements/frozen.txt --no-binary pycurl
```
Comment (RonnyPfannschmidt) [inline Dockerfile]: the abi issues with numpy/matplotlib are solved by adding `--no-binary numpy --no-binary matplotlib`

Comment (Fryguy) [reply Dockerfile]: Thank you!

**Comment by dajoRH** [comment]
Lint report for commit 036fdd1b942b8222e09ffbc6e4977f7d79d53a86:

`cfme/tests/ansible/test_embedded_ansible_actions.py`:
- :bangbang: Line 25:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 136:1: **E265** *block comment should start with '# '*

`cfme/tests/ansible/test_embedded_ansible_services.py`:
- :bangbang: Line 24:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/automate/test_buttons.py`:
- :bangbang: Line 137:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_class.py`:
- :bangbang: Line 97:1: **E265** *block comment should start with '# '*

`cfme/tests/automate/test_vmware_methods.py`:
- :bangbang: Line 81:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_object_tag_visibility.py`:
- :bangbang: Line 93:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud/test_cloud_timelines.py`:
- :bangbang: Line 131:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_events.py`:
- :bangbang: Line 37:1: **E265** *block comment should start with '# '*
- :bangbang: Line 38:1: **E265** *block comment should start with '# '*
- :bangbang: Line 41:1: **E265** *block comment should start with '# '*
- :bangbang: Line 42:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_rest_providers.py`:
- :bangbang: Line 208:1: **E265** *block comment should start with '# '*
- :bangbang: Line 230:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_retirement.py`:
- :bangbang: Line 194:1: **E265** *block comment should start with '# '*
- :bangbang: Line 228:1: **E265** *block comment should start with '# '*

`cfme/tests/cloud_infra_common/test_vm_ownership.py`:
- :bangbang: Line 19:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/configure/test_access_control.py`:
- :bangbang: Line 254:1: **E265** *block comment should start with '# '*
- :bangbang: Line 255:1: **E265** *block comment should start with '# '*
- :bangbang: Line 865:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_default_filters.py`:
- :bangbang: Line 60:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_log_depot_operation.py`:
- :bangbang: Line 187:1: **E265** *block comment should start with '# '*
- :bangbang: Line 215:1: **E265** *block comment should start with '# '*
- :bangbang: Line 235:1: **E265** *block comment should start with '# '*
- :bangbang: Line 237:1: **E265** *block comment should start with '# '*
- :bangbang: Line 289:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_register_appliance.py`:
- :bangbang: Line 116:1: **E265** *block comment should start with '# '*
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/configure/test_tag.py`:
- :bangbang: Line 202:5: **E265** *block comment should start with '# '*

`cfme/tests/containers/test_containers_summary.py`:
- :bangbang: Line 26:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_actions.py`:
- :bangbang: Line 44:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_alerts.py`:
- :bangbang: Line 35:1: **E122** *continuation line missing indentation or outdented*
- :bangbang: Line 36:1: **E122** *continuation line missing indentation or outdented*

`cfme/tests/control/test_compliance.py`:
- :bangbang: Line 193:1: **E265** *block comment should start with '# '*

`cfme/tests/control/test_rest_control.py`:
- :bangbang: Line 156:5: **E265** *block comment should start with '# '*
- :bangbang: Line 189:5: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_config_management.py`:
- :bangbang: Line 14:15: **E114** *indentation is not a multiple of four (comment)*
- :bangbang: Line 14:15: **E116** *unexpected indentation (comment)*
- :bangbang: Line 14:15: **E265** *block comment should start with '# '*

`cfme/tests/infrastructure/test_instance_analysis.py`:
- :bangbang: Line 610:1: **E265** *block comment should start with '# '*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**